docs/concepts.md

```diff
@@ -0,0 +1,94 @@
+<!--[metadata]>
++++
+title = "Machine concepts and help"
+description = "Understand concepts for Docker Machine, including drivers, base OS, IP addresses, environment variables"
+keywords = ["docker, machine, amazonec2, azure, digitalocean, google, openstack, rackspace, softlayer, virtualbox, vmwarefusion, vmwarevcloudair, vmwarevsphere, exoscale"]
+[menu.main]
+parent="workw_machine"
+weight=3
++++
+<![end-metadata]-->
+
+
+# Understand Machine concepts and get help
+
+Docker Machine allows you to provision Docker on virtual machines that reside
+either on your local system or on a cloud provider. Docker Machine creates a
+host on a VM and you use the Docker Engine client as needed to build images and
+create containers on the host.
+
+## Drivers for creating virtual machines
+
+To create a virtual machine, you supply Docker Machine with the name of the
+driver you want use. The driver represents the virtual environment. For example,
+on a local Linux, Mac, or Windows system the driver is typically Oracle Virtual
+Box. For cloud providers, Docker Machine supports drivers such as AWS, Microsoft
+Azure, Digital Ocean and many more. The Docker Machine reference includes a
+complete [list of the supported drivers](drivers/index.md).
+
+## Default base operating systems for local and cloud hosts
+
+Since Docker runs on Linux, each VM that Docker Machine provisions relies on a
+base operating system. For convenience, there are default base operating
+systems. For the Oracle Virtual Box driver, this base operating system is the
+`boot2docker.iso`. For drivers used to connect to cloud providers, the base
+operating system is Ubuntu 12.04+. You can change this default when you create
+a machine. The Docker Machine reference includes a complete [list of the
+supported operating systems](drivers/os-base.md).
+
+## IP addresses for Docker hosts
+
+For each machine you create, the Docker host address is the IP address of the
+Linux VM. This address is assigned by the `docker-machine create` subcommand.
+You use the `docker-machine ls` command to list the machines you have created.
+The `docker-machine ip <machine-name>` command returns a specific host's IP
+address.
+
+## Configuring CLI environment variables for a Docker host
+
+Before you can run a `docker` command on a machine, you configure your
+command-line to point to that machine. The `docker-machine env <machine-name>`
+subcommand outputs the configuration command you should use. When you run a
+container on the Docker host, the container's ports map to ports on the VM.
```

Comment:
There is also a `generic` driver, which allows provisioning a physical machine. So this would change to:
"the container's ports map to ports on the machine".